lms/views/api/canvas/sync.py

```diff
@@ -39,32 +41,5 @@ def sync(self):
         authority = self._request.registry.settings["h_authority"]
         return [group.groupid(authority) for group in groupings]
 
-    @property
-    def _is_speedgrader(self):
-        return "learner" in self._request.json
-
-    def group_set(self):
-        if self._is_speedgrader:
-            return int(self._request.json["learner"].get("group_set"))
-
-        return int(self._request.json["course"].get("group_set"))
-
-    @property
-    def _is_group_launch(self):
```

Comment:
If we have a group_set_id, it's a group launch.

Reply:
There's a bit of weirdness around having `group_set_id` be referred to as `group_set`.